internals-js/src/federation.ts

```diff
@@ -488,6 +480,48 @@ function validateInterfaceRuntimeImplementationFieldsTypes(
   }
 }
 
+function validateShareableNotRepeatedOnSameDeclaration(
+  element: ObjectType | FieldDefinition<ObjectType>,
+  metadata: FederationMetadata,
+  errorCollector: GraphQLError[],
+) {
+  const shareableApplications: Directive<any, {}>[] = element.appliedDirectivesOf(metadata.shareableDirective());
```

Comment:
Is there a reason you want `Directive<any, {}>[]` instead of `Directive[]`?

Reply:
It was more than the former was a bit more precise than the later (making it clear there is not argument to the applications), but certainly doesn't really change anything to that method so happy to shorten it.